fetch/client/src/main/scala/endpoints4s/fetch/TextDecoder.scala

```diff
@@ -0,0 +1,11 @@
+package endpoints4s.fetch
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSGlobal
+import scala.scalajs.js.typedarray.ArrayBufferView
+
+@js.native
+@JSGlobal
+class TextDecoder(utfLabel: js.UndefOr[String] = js.undefined) extends js.Object {
```

Comment:
Should we make this private? Or `private[fetch]`?

Reply:
Will remove, no longer needed because `ChunkedEntities` will not be implemented.